<!-- -->
<launch>


<!--read waypoints and publish next waypoint-->
	<node pkg="goal_manager" type="robot_goal_manager" name="robot_goal_mangager" output="screen">
		<param name="next_point" value="/point"/>
		<param name="next_pose" value="/pose"/>
		<param name="waypoint_list" value="$(find goal_manager)/waypoints/global_goals.txt"/>
		<param name="path_publisher" value="/frame_path"/>
		<!--as lower frequency as longer the robot will wait at a reached goalpoint-->
		<param name="frequency" value="2"/>
		<param name="user_goal_input" value="/move_base_simple/goal"/>
		<param name="rabbit_frame" value="goal_point"/>
		<param name="robot_frame" value="base_link"/>
		<param name="fixed_frame" value="odom"/>
		<param name="waittime" value="0.1"/>
		<param name="wait_for_topic" value="true"/>
		<param name="wait_topic_name" value="/movement_finished"/>
		
	</node>
</launch>
